benchmarks/operator_benchmark/pt/add_test.py

```diff
@@ -6,8 +6,84 @@
 import operator_benchmark as op_bench
 import torch
 
+from torch.utils._benchmark.op_fuzzers import binary
+
 """Microbenchmarks for add_ operator. Supports both Caffe2/PyTorch."""
 
+
+def make_fuzzed_config(
+    scale=binary.SMALL,
+    n: int = 10,
+    seed: int = 0,
+    cross_product_configs=None,
+    tags=None,
+    checksum=None):
+
+    fuzzer = binary.BinaryOpFuzzer(seed=seed, scale=scale)
+    attr_names=["x_size", "y_size"]
```

Comment:
That's a good point. (Technically those are from the binary op fuzzer, but the fact that I'm saying "the magic hard coded string comes from X rather than Y kind of proves your point. ^^) I added attributes `X_SIZE` and `Y_SIZE` to binary, and switched the add benchmark to use them instead.